package.json

```diff
@@ -1,12 +1,22 @@
 {
   "name": "@balena/open-balena-api",
-  "private": true,
   "description": "Internet of things, Made Simple",
   "version": "0.0.1",
   "repository": {
     "type": "git",
     "url": "https://github.com/balena-io/open-balena-api"
   },
+  "main": "dist/index",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build": "npm run clean && tsc --project . && copyup \"src/**/*.sbvr\" \"src/**/*.sql\" dist/",
```

Comment:
Is `copyup` from this `copyfiles` dep?

Reply:
Yes, `copyup` is a "binary" equivalent to `copyfiles --up 1 ...`. I could change back to that if you think it's clearer.

Reply:
No it's fine, I just wasn't sure if it was some mac binary or not